plugin/src/main/java/org/opensearch/sql/plugin/config/OpenSearchPluginModule.java

```diff
@@ -100,13 +100,12 @@ public SQLService sqlService(QueryManager queryManager, QueryPlanFactory queryPl
    */
   @Provides
   public QueryPlanFactory queryPlanFactory(DataSourceService dataSourceService,
-      ExecutionEngine executionEngine) {
-    Analyzer analyzer =
-        new Analyzer(
-            new ExpressionAnalyzer(functionRepository), dataSourceService, functionRepository);
+                                           ExecutionEngine executionEngine,
+                                           Settings settings) {
```

Comment:
Why are these two rows indented so much, does it have to do with checkstyle?

Reply:
Both styling options match checkstyle (for second+ arguments):
1. +4 from function indent
2. on the same level as first arg

Reply:
To improve you can put the first parameter on a new line so that all parameters are on one indentation only.